scripts/firewall/rules.v6

```diff
@@ -2,21 +2,34 @@
 :INPUT ACCEPT [0:0]
 :FORWARD ACCEPT [0:0]
 :OUTPUT ACCEPT [0:0]
+:CJDNS - [0:0]
+:YGGDRASIL - [0:0]
 :YGGCLIENT - [0:0]
 -A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
 -A INPUT -p ipv6-icmp -j ACCEPT
 -A INPUT -i lo -j ACCEPT
--A INPUT -j ACCEPT -p tcp --dport 22
--A INPUT -j ACCEPT -p tcp --dport 80
--A INPUT -j ACCEPT -p tcp --dport 443
--A INPUT -j ACCEPT -p tcp --dport 3000
--A INPUT -j ACCEPT -p tcp --dport 5201
--A INPUT -j ACCEPT -p tcp --dport 4001
--A INPUT -j ACCEPT -p tcp --dport 9090
--A INPUT -j ACCEPT -p tcp --dport 9100
--A INPUT -j ACCEPT -p udp --dport 12345
+-A INPUT -j ACCEPT -p tcp --dport 12345
 -A INPUT -j ACCEPT -p udp --dport 9001 -d ff02::114
```

Comment:
If memory serves this is the YGGDRASIL announcement for peering. Blocking this will prevent yggdrasil from automatically peering

FF02::114 is a Local Link multicast address (think of it as a broadcast)

`FF02:0:0:0:0:0:0:100-FF02:0:0:0:0:0:0:15E - variable scope allocation`